src/internal/connector/exchange/data_collections.go

```diff
@@ -55,6 +55,16 @@ func (dps DeltaPaths) AddPath(k, p string) {
 	dps[k] = dp
 }
 
+// RemoveDelta removes the delta from the DeltaPaths.
+// It is *ONLY* for use in tests.
+func (dps DeltaPaths) RemoveDelta(k string) {
```

Comment:
This should be moved to a test file.  There's no reason for it to exist within the primary package if its only used for testing.
```suggestion
func RemoveDelta(t *testing.T, dps DeltaPaths, k string) {
```

Reply:
This function is used in another package and the values in DeltaPaths were private. I'm converting them to public and moving this out to the test package.